components/style/properties/gecko.mako.rs

```diff
@@ -3152,6 +3142,14 @@ fn static_assert() {
         self.copy_display_from(other)
     }
 
+    pub fn set_adjusted_display(
+        &mut self,
+        v: longhands::display::computed_value::T,
+        _is_item_or_root: bool
+    ) {
+        self.gecko.mOriginalDisplay = Self::match_display_keyword(v);
```

Comment:
This is still not what needs to happen?

So, to clarify, `mOriginalDisplay` can't change from `StyleAdjuster`, only `mDisplay` should. This does the opposite as written.

Reply:
Ahh, yeah, sorry, I miss this one 😣
Please let me update it again!